pkg/analytics/track.go

```diff
@@ -407,3 +413,13 @@ func track(event string, success bool, props map[string]interface{}) {
 		oktetoLog.Infof("Failed to send analytics: %s", err)
 	}
 }
+
+func disabledByOktetoAdmin() bool {
+	if !okteto.IsOkteto() {
+		return false
+	}
+	if okteto.IsOktetoCloud() {
```

Comment:
Can we remove the dependency on cloud and instead depend on what the context is telling us?

Reply:
No, `telemetry` is disabled in Cloud, and we don't want to lose all `okteto up`s happening in Cloud

Reply:
Can we enable telemetry in cloud? Every "if cloud do x" block adds complexity and the possibilities to more bugs.

Reply:
That's has deeper implications is out of the scope of this PR